zendesk/user.go

```diff
@@ -128,6 +129,8 @@ type UserAPI interface {
 	GetOrganizationUsersIterator(ctx context.Context, opts *PaginationOptions) *Iterator[User]
 	GetOrganizationUsersOBP(ctx context.Context, opts *OBPOptions) ([]User, Page, error)
 	GetOrganizationUsersCBP(ctx context.Context, opts *CBPOptions) ([]User, CursorPaginationMeta, error)
+	GetGroupUsersOBP(ctx context.Context, opts *OBPOptions) ([]User, Page, error)
```

Comment:
This is missing the generated `GetGroupUsersIterator(ctx context.Context, opts *PaginationOptions) *Iterator[User]`